ecosystem.cfg

```diff
@@ -55,6 +56,11 @@ Dexterity-ecosystem =
     plone.formwidget.autocomplete
     plone.formwidget.contenttree
     z3c.relationfield
+Grok =
```

Comment:
@thet I think this line is not needed, although it can be fixed at any time later on. Am I right @tisto ?